src/Template/Element/advanced_search_form.ctp

```diff
@@ -30,6 +30,21 @@ echo $this->Form->create('AdvancedSearch', [
 ?>
 
 <div layout="column" ng-app="app" ng-cloak>
+    <?php if ($this->request->getParam("?") && !(isset($isSidebar) && $isSidebar)): ?>
+        <div id="template-notif">
+            <i><?= __("This form has been filled by a template.")?></i>
+            <md-button class="md-primary md-raised"
+                        href="<?= $this->Url->build([
+                            "controller" => "Sentences",
+                            "action" => "advanced_search",
+                            "?" => [],
+                        ]);?>">
+                <?= __('Clear Form') ?>
```

Comment:
About the button label, we use sentence case in the rest of the website, so please use `Clear form` instead.